modules/integration/tests-integration/tests-service/src/test/java/org/wso2/carbon/ml/dataset/test/CreateExternalDatasetsTestCase.java

```diff
@@ -56,6 +60,8 @@ public void testCreateDatasetDigits() throws IOException {
         try {
             response = mlHttpclient.uploadDatasetFromCSV(MLIntegrationTestConstants.DATASET_NAME_DIGITS,
                     "1.0", MLIntegrationTestConstants.DIGIT_RECOGNITION_DATASET_SAMPLE);
+            assertEquals("Unexpected response received", Response.Status.OK.getStatusCode(), response.getStatusLine()
+                    .getStatusCode());
```

Comment:
The exact files do not exist now. Fixed the existing files in PR: https://github.com/wso2/product-ml/pull/262